web-api/src/persistence/elasticsearch/getPractitionersByName.ts

```diff
@@ -1,19 +1,27 @@
 import { IS_PRACTITIONER } from './helpers/searchClauses';
-import { MAX_SEARCH_CLIENT_RESULTS } from '../../../../shared/src/business/entities/EntityConstants';
+import { PRACTITIONER_SEARCH_PAGE_SIZE } from '@shared/business/entities/EntityConstants';
+import { PractitionerSearchResultType } from '@web-client/presenter/computeds/AdvancedSearch/practitionerSearchHelper';
```

Comment:
Hmmm are we sure we want presenter types imported here in the backend?

I am totally open to the idea that I'm wrong, just don't want to tightly couple our persistence to our presenter.

Reply:
Is there already a place for shared types? Nothing jumping out at me in the `shared` directory.

Where's a better place for this type to live?

Reply:
I think coming up with a shared place is a good idea, I don't want to dismiss that at all 😄  

I think my original point was more that the type the presenter is using to render the view is, _in most cases_, different than what persistence is returning anyways. For example, the type here isn't accurate. `getPractitionersByName` is typed as returning a list of `PractitionerSearchResultType` - which includes the property `formattedAdmissionsDate` which is computed in the helper. To me, it seems like this type should be what is included in `_source`, since that is what is being pulled for each matching record in OpenSearch

Reply:
I created a new type in `getPractitionersByName` and changed the name of the old `PractitionerSearchResultType`. let me know what you think!